src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/5.diff.sql

```diff
@@ -0,0 +1,1613 @@
+/****************************************************************************************
+Guidelines to create migration scripts - https://github.com/microsoft/healthcare-shared-components/tree/master/src/Microsoft.Health.SqlServer/SqlSchemaScriptsGuidelines.md
+******************************************************************************************/
+SET XACT_ABORT ON
+
+BEGIN TRANSACTION
+
+---BEGIN - CREATE/ALTER Statements----------
+
+IF NOT EXISTS (
+    SELECT *
+    FROM sys.tables
+    WHERE name = 'Partition')
+BEGIN
+    CREATE TABLE dbo.Partition (
+        PartitionKey            BIGINT                 NOT NULL, --PK
```

Comment:
nit: should we had some comments on difference between PartitionKey and PartitionId

Reply:
Also is BIGINT necessary? Maybe INT? Will save four bytes on each row

Reply:
Sure, INT makes sense

Reply:
Added comments and changed the type to INT.